app/src/main/java/agency/five/tmdb/MovieDatabaseImpl.kt

```diff
@@ -0,0 +1,28 @@
+package agency.five.tmdb
+
+import agency.five.tmdb.ui.components.MovieItemViewState
+import android.util.Log
+
+class MovieDatabaseImpl() : MovieDatabase {
+    private val favoriteMovies = mutableListOf<MovieItemViewState>()
```

Comment:
You should also have an additional model for the database. Currently, this change is not necessary as you will need to create a new model once the actual database starts to be used :)

I want you to pay attention to the correct implementation of the architecture.
For more details, ask the lecturers who will explain it to you :)